automation_tools/__init__.py

```diff
@@ -1823,16 +1799,15 @@ def apply_hotfix():
 
 def upstream_install(admin_password=None, run_katello_installer=True):
     """Task to install Foreman nightly using forklift scripts"""
-    koji = 'koji' in os.environ.get('DISTRIBUTION').lower()
+    koji = 'koji' in os.environ.get('SATELLITE_DISTRIBUTION').lower()
```

Comment:
We are exporting SATELLITE_DISTRIBUTION in installer and other jobs, are you going to handle this in any PR?

Reply:
Yeah, so I'm fixing it ```s/DISTRIBUTION/SATELLITE_DISTRIBUTION/```

@ntkathole you seem to me you misread the direction (+/- lines)

Reply:
Thanks you made me think and I improved the code to:
```
koji = 'koji' in os.environ.get('SATELLITE_DISTRIBUTION','').lower()
```
```''.lower()``` is much better than ```None.lower()``` 
Now it will work even if there is no ```SATELLITE_DISTRIBUTION``` defined ;-)

Reply:
@lpramuk we have DISTRIBUTION exported in satellite-installer job but not SATELLITE_DISTRIBUTION

Reply:
@ntkathole you are right we have spaghetti code in ```satellite6-provisioning.sh```

https://github.com/SatelliteQE/robottelo-ci/blob/master/scripts/satellite6-provisioning.sh#L33-L49